packages/editor/playground/playground.tsx

```diff
@@ -94,35 +124,50 @@ const App: FC = () => {
     const initialContentId = params.get("contentid")
     const enableDevTools = params.get("devtools") !== "false"
 
-    const note = useMemo(() => {
-        let content = defaultContent
-        if (isString(initialContent)) {
-            content = initialContent
-        } else if (initialContentId === "long") {
-            content = longContent
-        } else if (initialContentId === "just-code") {
-            content = justCodeContent
-        }
+    const [hasUnsavedChanges, setHasUnsavedChanges] = useState(false)
 
+    if (isString(initialContent) && initialContent.length > 0) {
+        contentMap["customize"] = initialContent
+    }
+    let defaultContentId = "default"
+    if (initialContentId !== null && initialContentId in contentMap) {
+        defaultContentId = initialContentId
+    }
+
+    const [contentId, setContentId] = useState(defaultContentId)
+    const [content, setContent] = useState(contentMap[defaultContentId])
+
+    const note = useMemo(() => {
         return {
             content,
             deleted: false,
         }
-    }, [initialContent, initialContentId])
+    }, [content])
 
-    const [hasUnsavedChanges, setHasUnsavedChanges] = useState(false)
-    const [content, setContent] = useState("")
+    const changeContentSelect = function (e: string) {
+        console.log(e)
+        setContentId(e)
+        setContent(contentMap[e])
+    }
 
     return (
         <div style={{ display: "flex", flexDirection: "column" }}>
             <Editor
+                key={content}
```

Comment:
```suggestion
                key={contentId}
```

We won't want to rerender the `<Editor/>` whenever the `content` change.